tests/Unit/Parallel/Test_PupStlCpp11.cpp

```diff
@@ -5,19 +5,18 @@
 
 #include <algorithm>
 #include <array>
+#include <cstddef>
 #include <deque>
 #include <memory>
 #include <pup.h>
 #include <pup_stl.h>
-#include <stddef.h>
 #include <string>
 #include <tuple>
 #include <unordered_map>
 #include <unordered_set>
 #include <vector>
 
 #include "Parallel/CharmPupable.hpp"
-#include "Parallel/PupStlCpp11.hpp"
```

Comment:
Since the file is gone, do we still want to keep the tests for it even though the code has moved upstream?

Reply:
Good point. I think we should not.

Reply:
These tests already found a bug in Charm++'s code a while back, so I think we should test for it.